README.md

```diff
@@ -456,6 +474,21 @@ Then just launch task:
 ./gradlew :sample:mpp-library:debugFatFramework
 ```
 
+### Example 10 - plain file resource access
+The first step is a create a resource file `test.txt` for example, in `commonMain/resources/MR/files`
+After gradle sync we can get file by id `MR.files.test`
+Moko-resources has out of box implementation function for read text files from common code - `readText()`
+
+Usage on Android:
+```
+val text = MR.files.test.getText()
+```
+Usage on Apple:
+```
+val text = FileResource("test", "txt").readText()
```

Comment:
why here created resource? here should be `MR.files.test`

Reply:
fixed